requirements.txt

```diff
@@ -11,7 +11,7 @@ blinker==1.6.2
 cachetools==5.3.0
 certifi==2022.12.7
 charset-normalizer==3.1.0
-chromadb==0.3.21
+chromadb==0.3.29

```

Comment:
chromadb 0.3.21 had an issue with the langchain library for me. Bumping the version here made it work.